src/components/content/content.tsx

```diff
@@ -23,12 +23,12 @@ export const Content: React.FC<ContentProps> = ({ selectedNotesGroup }) => {
             }
             return false;
         });
-        updateStateNotesGroup({ ...notesGroup })
+        updateStateNotesGroup({ ...notesGroup } as NotesGroup)
```

Comment:
Thanks a lot for fixing the issues :)
Just wondering why is this change needed?